src/main/flight/pid.c

```diff
@@ -371,6 +395,14 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
             float DTerm = Kd[axis] * delta * tpaFactor;
             DEBUG_SET(DEBUG_DTERM_FILTER, axis, DTerm);
 
+            // if either crashDtermThreshold or crashGyroThreshold is non-zero then crash recovery is on, so check for a crash
+            if (crashDtermThreshold > 0 || crashGyroThreshold > 0) {
```

Comment:
It's probably best to used `delta`. However during testing using `DTerm` means people can directly compare the parameter with black box log output.